docs/modules/ROOT/pages/index.adoc

```diff
@@ -499,13 +504,15 @@ The `expected_claim_generator` value SHOULD contain a CBOR map with the followin
 If the *<<_identity_assertion,identity assertion>> generator* chooses to include the `expected_countersigners` value in `signer_payload`, it MUST be computed as follows:
 
 . For every _other_ *<<_identity_assertion,identity assertion>>* that will be present in the _<<C2PA Manifest>>:_
-.. Identify the signing credential that will be used to sign that *<<_identity_assertion,identity assertion>>.*
-.. Serialize the signing credential as a CBOR byte string as described in xref:_serialization_for_credential_types[]. This serialization should be performed using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
-.. Compute a hash of the serialized end-entity certificate using one of the algorithms documented in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_hashing_2++[Section 13.1, “Hashing,”] of the C2PA technical specification.
-.. An entry in `expected_countersigners` SHOULD be added containing the hash from the previous step and the corresponding `alg` value for the selected hash algorithm.
-. Once the signing credentials have been gathered, the `expected_countersigners` value SHOULD contain a CBOR array of CBOR maps with the following fields:
-.. `alg` describing the hash algorithm as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_hashing_2++[Section 13.1, “Hashing,”] of the C2PA technical specification, and
-.. `hash` value containing the computed hash of the serialized signing credential.
+.. Calculate the `signer_payload` value for that *<<_identity_assertion,identity assertion>>.*
```

Comment:
Almost ! The signer_payload of the other ID assertion cannot include the 'expected_countersigners' field as this gets us back into circular referencing difficulties. All other fields are ok. So I think this should be:
"Calculate the signer_payload value for that identity assertion, but omitting any instance of the optional 'expected_countersigners' field that may be present within that identity assertion signer_payload_map". All other fields must be included.

Maybe we should refer to this as the partial_signer_payload?